docs/test/misc/ds4/alignment-controls-grid.html

```diff
@@ -51,17 +51,15 @@ <h2>Medium</h2>
         </div>
         <div class="grid__cell">
             <span class="select">
-                <span class="select__control">
-                    <select class="select__control">
-                        <option value="1">Option 1</option>
-                        <option value="2">Option 2</option>
-                        <option value="3">Option 3</option>
-                    </select>
-                    <span class="select__icon" hidden>
-                        <svg aria-hidden="true" focusable="false">
-                            <use xlink:href="#svg-icon-arrow-down"></use>
-                        </svg>
-                    </span>
+                <select class="select__control">
+                    <option value="1">Option 1</option>
+                    <option value="2">Option 2</option>
+                    <option value="3">Option 3</option>
+                </select>
+                <span class="select__icon" hidden>
+                    <svg hidden="true" focusable="false">
```

Comment:
Same as above. `hidden`

Reply:
Fixed this by removing the unneeded svg completely